lib/style.css

```diff
@@ -101,6 +115,8 @@ body {
 }
 
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
```

Comment:
Yea, totally. Maybe we should add an inline `TODO` so we don't lose track of them?